themes/rebellion/Makefile

```diff
@@ -0,0 +1,6 @@
+PLUGIN_NAME=		rebellion
```

Comment:
I'm a bit worried about polluting the plugin namespace. how about "theme-rebellion" ?